Attempt to fix merging: remove all kinds of elements from implementation which have been moved to a group

benjamin-hg opened the pull request:
Couldn't test it, but this hopefully fixes https://github.com/0install/0repo/issues/34

bastianeicher commented:
Just tested this locally and it seems to fix the issue. :)

bastianeicher commented:
@talex5, could you merge this and publish a patch-level update? We'd like to use it in a CI setup and would prefer not to use a Git checkout with `0install add-feed`.

benjamin-hg commented:
> Three commits for a one-line change? ;-)
> 
> I've squashed them and added a unit-test for this at #37 - closing in favour of that...

I tried to get the unit tests running on my windows machine (first time I did anything with python), but finally gave up. Would have loved to run and test my fix locally though.

Thanks for merging!
